src/components/general/OverflowMenu/OverflowMenu.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
```

Comment:
This entire file (along with index.tsx) can be deleted

Reply:
Romoved Overflow stories and index.tsx.